src/borg/archiver.py

```diff
@@ -2424,6 +2471,10 @@ def process_epilog(epilog):
         subparser.add_argument('--format', '--list-format', dest='format', type=str,
                                help="""specify format for file listing
                                 (default: "{mode} {user:6} {group:6} {size:8d} {isomtime} {path}{extra}{NL}")""")
+        subparser.add_argument('--json', action='store_true',
+                               help='format output as JSON. The form of --format is ignored, but keys used in it '
+                                    'are added to the JSON output. Some keys are always present. Note: JSON can only '
+                                    'represent text. A "bpath" key is therefore not available.')
```

Comment:
Other commands don't have that, so less consistent.

What could make more sense imho is to have --format just list (as in `name,sha256,mtime`) the keys wanted, and then only give back these keys.